src/main/java/org/opensearch/ad/AnomalyDetectorJobRunner.java

```diff
@@ -134,12 +136,12 @@ public void runJob(ScheduledJobParameter scheduledJobParameter, JobExecutionCont
         String detectorId = scheduledJobParameter.getName();
         log.info("Start to run AD job {}", detectorId);
         adTaskManager.refreshRealtimeJobRunTime(detectorId);
-        if (!(scheduledJobParameter instanceof AnomalyDetectorJob)) {
+        if (!(scheduledJobParameter instanceof Job)) {
             throw new IllegalArgumentException(
                 "Job parameter is not instance of AnomalyDetectorJob, type: " + scheduledJobParameter.getClass().getCanonicalName()
```

Comment:
nit: Message here should probably change to "instance of AnomalyDetectorJob or ForecasterJob" or Just analysis/timeseries job?

Reply:
Changed to "Job parameter is not instance of Job," as forecasting and AD share the same Job class.